src/SmartTransactionsController.test.ts

```diff
@@ -450,29 +502,53 @@ describe('SmartTransactionsController', () => {
   });
 
   describe('submitSignedTransactions', () => {
-    beforeEach(() => {
-      // eslint-disable-next-line jest/prefer-spy-on
-      smartTransactionsController.checkPoll = jest.fn(() => ({}));
-    });
-
     it('submits a smart transaction with signed transactions', async () => {
-      const signedTransaction = createSignedTransaction();
-      const signedCanceledTransaction = createSignedCanceledTransaction();
-      const submitTransactionsApiResponse = createSubmitTransactionsApiResponse(); // It has uuid.
-      nock(API_BASE_URL)
-        .post(`/networks/${ethereumChainIdDec}/submitTransactions`)
-        .reply(200, submitTransactionsApiResponse);
-
-      await smartTransactionsController.submitSignedTransactions({
-        signedTransactions: [signedTransaction],
-        signedCanceledTransactions: [signedCanceledTransaction],
-        txParams: signedTransaction,
-      });
-
-      expect(
-        smartTransactionsController.state.smartTransactionsState
-          .smartTransactions[CHAIN_IDS.ETHEREUM][0].uuid,
-      ).toStrictEqual('dP23W7c2kt4FK9TmXOkz1UM2F20');
+      const chainIdHex = '0x64';
+      const chainIdDec = 100;
```

Comment:
Nit: `chainIdDec` is a number, not decimal
```suggestion
      const chainId = 100;
      const chainIdHex = `0x${chainId.toString(16)}`; // 0x64
```

Reply:
Oh sure that works!